frontend/src/utils/helpers.ts

```diff
@@ -169,3 +175,45 @@ export const displayDateInfo = (date: Date) => {
 export const formatRoleOrStatus = (str: string) => {
   return str[0] + str.substring(1).toLowerCase();
 };
+
+/**
+ * Upload image to firebase
+ * @param userID          - The userID of who uploads the image
+ * @param image           - The image file to upload
+ * @returns The string URL to the event.
+ */
+export const uploadImage = async (userID: string | null, image: File) => {
+  if (!userID) {
+    console.error("No eventId provided");
+    return "";
+  }
+
+  const fileExtension = image.name.slice(image.name.lastIndexOf("."));
+  const currentTime = Date.now();
+  const refString = `${userID}_${currentTime}${fileExtension}`;
+  const storage = getStorage();
+  const imageRef = ref(storage, refString);
+  const uploadTask = uploadBytesResumable(imageRef, image);
+
+  // Retrieve the download URL
+  try {
+    // Wait for the upload to complete and retrieve the download URL
+    await uploadTask.then(); // Ensures the upload completes before proceeding
```

Comment:
I'm not sure a `.then()` does anything here if it is already wrapped in a try catch block. Why are you using it?